src/shared/ui/survey/CheckBox/CheckBox.item.tsx

```diff
@@ -71,7 +71,7 @@ const CheckBoxItem: FC<Props> = ({
             {!!tooltip && (
               <Tooltip
                 markdown={tooltipText}
-                accessibilityLabel="checkbox-tooltip-view"
+                accessibilityLabel={`checkbox-tooltip-view-${tooltipText}`}
```

Comment:
Please make sure that nothing breaks if `tooltipText` contains markdown

Reply:
it's ok, tried many special char-s